src/auth/screens/events.screen.js

```diff
@@ -149,7 +149,7 @@ class Events extends Component {
       case 'MemberEvent':
         return `${userEvent.payload.action}`;
       case 'PublicEvent':
-        return translate('auth.events.issuesEvent', language);
+        return translate('auth.events.publicEvent', language);
```

Comment:
@machour I did this (`{{user}} made {{repo}} public`), but in PR #193, however you can copy the [changes](https://github.com/gitpoint/git-point/pull/193/commits/1c2aa767e6cd0cd2168620f780d55135367f3301) (at the same time, you can come up with a better solution than mine)  and move it to this PR.